docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/eks-helm.mdx

```diff
@@ -0,0 +1,360 @@
+
+The following will install `external-dns` in the `external-dns` namespace via Helm. For more configuration options, conduct the [Helm chart](https://github.com/kubernetes-sigs/external-dns/tree/master/charts/external-dns).
+
+Consider setting `domainFilters` via `--set` to restrict access to certain hosted zones.
+
+```shell
+helm upgrade --install \
+  external-dns external-dns \
+  --repo https://kubernetes-sigs.github.io/external-dns/ \
+  --version $EXTERNAL_DNS_HELM_CHART_VERSION \
+  --set "env[0].name=AWS_DEFAULT_REGION" \
+  --set "env[0].value=$REGION" \
+  --set txtOwnerId=external-dns \
+  --set policy=sync \
+  --set "serviceAccount.annotations.eks\.amazonaws\.com\/role-arn=$EXTERNAL_DNS_IRSA_ARN" \
+  --namespace external-dns \
+  --create-namespace
+```
+
+#### cert-manager
+
+Cert-manager is an open-source Kubernetes add-on that automates the management and issuance of TLS certificates. It integrates with various certificate authorities (CAs) and provides a straightforward way to obtain, renew, and manage SSL/TLS certificates for your Kubernetes applications.
+
+The following will install `cert-manager` in the `cert-manager` namespace via Helm. For more configuration options, conduct the [Helm chart](https://artifacthub.io/packages/helm/cert-manager/cert-manager). The supplied settings will also configure `cert-manager` to ease the certificate creation by setting a default issuer, which allows us to add a single annotation on an ingress to request the relevant certificates.
+
+```shell
+helm upgrade --install \
+  cert-manager cert-manager \
+  --repo https://charts.jetstack.io \
+  --version $CERT_MANAGER_HELM_CHART_VERSION \
+  --namespace cert-manager \
+  --create-namespace \
+  --set installCRDs=true \
+  --set "serviceAccount.annotations.eks\.amazonaws\.com\/role-arn=$CERT_MANAGER_IRSA_ARN" \
+  --set securityContext.fsGroup=1001 \
+  --set ingressShim.defaultIssuerName=letsencrypt-production \
+  --set ingressShim.defaultIssuerKind=ClusterIssuer \
+  --set ingressShim.defaultIssuerGroup=cert-manager.io
+```
+
+It is recommended to install the cert-manager custom resource definitions (CRDs) separately from Helm. Please refer to the [installation advice](https://cert-manager.io/docs/installation/helm/#crd-installation-advice) provided by cert-manager. This approach is recommended because when using Helm, there is a possibility of accidentally omitting the CRDs due to incorrect configuration resulting the loss of certificates.
+
+Create a ClusterIssuer via `kubectl` to enable cert-manager to request certificates from [Let's Encrypt](https://letsencrypt.org/).
+
+```shell
+cat << EOF | kubectl apply -f -
+---
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-production
```

Comment:
maybe drop `production`? This is the first time we have anything about prod mentioned and I think it may be confusing:
```suggestion
  name: letsencrypt
```